Wonderfulboyx submission

wonderfulboyx opened the pull request:
Wonderfulboyx PR

pybites commented:
Cool, thanks, we want to learn Django, so nice to see you used it for this challenge. Nice little app.
 
Couple of things I noted (might be missing something as I am new to Django) and changed after merge on our Community branch: 

- pip install -r requirements.txt -> could not find mysql-connector-python so commented it out

- I deleted .idea and *sqlite3 from version control (.gitignore)

- I documented entry point of app in README: 
    - python manage.py migrate
    - python manage.py runserver

- After running migrate (new sqlite3) company and device select boxes are empty, where can I add more companies and devices (in urls I only see / and /show routes)

- mysite/settings.py seems to expose SECRET_KEY, maybe you want to load this from environment (os.environ.get)?

Thanks,
Bob

pybites commented:
one other thing: requirements.txt (46 lines) contains a lot of unrelated stuff, maybe you could clean it up?

wonderfulboyx commented:
Thank you for comment and your generate welcome.

I modified my app
- added feature that create device and company
- cleaned up requirements.txt
- hid SECRET_KEY

I will create new pull request
